src/main/java/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -4,12 +4,14 @@
 import jakarta.validation.constraints.PositiveOrZero;
 import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
+import java.util.List;
 
 public record CreateBookRequestDto(@NotNull String title,
                                    @NotNull String author,
                                    @NotNull @Size(min = 5, max = 10)
                                    String isbn,
                                    @NotNull @PositiveOrZero BigDecimal price,
                                    String description,
-                                   String coverImage) {
+                                   String coverImage,
+                                   List<Long> categoryIds) {
```

Comment:
you may also add validation here
```suggestion
                                   @NotEmpty List<@Positive Long> categoryIds) {
```